### DISCUSSION TITLE
Can it parse .brd file now?

### DISCUSSION THREAD

**Discussion by wbycloud** [Werni2A/OpenAllegroParser#13] [Q&A]
Hi!

I want to parse Allegro's brd file，but I didn't find any soft can parse it.

Can the OpenAllegroParser is able to do it now?

Thank you!

**Comment by Werni2A** [comment]
In short, OpenAllegroParser is unable to parse Allegro's `brd` files at the moment.

I plan on implementing a `brd` parser at some point, however this requires a lot more effort and time. This will probably take another one or two years until the parser can handle `brd` files.
